simple-podcasting.php

```diff
@@ -251,6 +251,54 @@ function setup_edit_screen() {
 }
 add_action( 'admin_init', __NAMESPACE__ . '\setup_edit_screen' );
 
+/**
+ * Registers block assets for Podcast block.
+ */
+function register_podcast_block_assets() {
+	$block_asset = [
+		'version'      => PODCASTING_VERSION,
+		'dependencies' => [],
+	];
+	if ( file_exists( PODCASTING_PATH . 'dist/podcast.asset.php' ) ) {
+		$block_asset = require PODCASTING_PATH . 'dist/podcast.asset.php';
```

Comment:
The asset file includes the JavaScript dependencies rather than the CSS dependencies.

I think you'll need to define them manually for now as some of them don't exist as CSS registrations (lodash for example).

Reply:
@peterwilsoncc @jeffpaul Little confused by the comment on this one. Is the concern here we're using the wrong asset file?

Reply:
@Firestorm980  This can be ignored.

I misread the code that follows and thought it was using the `$block_asset['dependencies']` when enqueing the stylesheet but was mistaken. 

Sorry